packages/fcl/src/current-user/service-of-type.js

```diff
@@ -1,3 +1,30 @@
+import {invariant} from "@onflow/util-invariant"
+
+function compareVersions(a, b) {
```

Comment:
I think we have a few versions of this helper floating around across projects => @chasefleming has one for FCL Discovery as well I think. Can we split this out into a separate util and share it cross projects?

Reply:
I think this is the one @chasefleming  has: https://github.com/onflow/fcl-discovery/blob/master/helpers/version.ts#L3

Reply:
Yeah, I can break it out into a `util-semver` package